source/use-cases/client-side-field-level-encryption-guide.txt

```diff
@@ -158,12 +160,22 @@ compliant to data privacy regulations with MongoDB.
 Implementation
 --------------
 
-This guide is meant for users new to Client-Side Field Level Encryption (CSFLE).
-It covers the why and how at a high level, and is not meant to produce working
-code in a single file.
+This section explains the following configuration and implementation
+details of CSFLE:
+
+- Software required to run your client and server in your local development
+  environment.
+
+- Creation, setup, and validation of the encryption keys.
+
+- Configuration of the client for automatic field-level encryption.
+
+- Demonstration of the client writing, querying, and reading encrypted
+  fields.
 
-To view and run minimal end-to-end examples based on this guide, refer to our
-`CSFLE guides <https://github.com/mongodb-university/csfle-guides>`_ repository.
+For complete runnable examples of CSFLE based on this scenario, see the
+`CSFLE guides <https://github.com/mongodb-university/csfle-guides>`_
```

Comment:
In the new note above we refer readers to the **Download Example Project** section. Should we mirror that here or remove this entirely?

Reply:
I voted to remove this, but I think @terakilobyte wanted to keep it. Any additional thoughts Nathan?

Reply:
Let's remove and see how it performs with user feedback